api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>I think this text should be updated since LST IDs are set a little further down.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>